.github/script/call-canary-deploy-demo-prod

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+const { spawnOrFail, logger } = require('../../script/cli-utils');
+
+const myArgs = process.argv.splice(2);
+var demoCurrentVersion = myArgs[0];
```

Comment:
This should be deployed version (e.g. 2.28 or 3.0.0 etc).

Reply:
Changed this variable name to `demoDeployVersion`.